RereleaseTimer/appcenter-api.js

```diff
@@ -0,0 +1,86 @@
+const request = require('request-promise');
+const utils = require('./utils');
+const token = process.env["APP_CENTER_TOKEN"];
+
+module.exports = {
```

Comment:
How frequently are we going to call this function? Because if one of the request is failed - we will need to wait for additional 30 minutes or so, this way I believe we need to implement some kind of request retry logic.

Reply:
It's every minute, not 30